### ISSUE TITLE
audio seeker not working

### ISSUE THREAD

**Issue by akhil-221** [bluefireteam/audioplayers#1203]
i have used audioplayer package and its seeks functionality not working in android real device,but when i used with IOS simulator it works properly

**Comment by abhinavkulkarni** [comment]
Yes, I can confirm that I am not getting any events in the `onSeekComplete` stream, however I do in the other streams. This is on the web, haven't checked on other platforms.

**Comment by Gustl22** [comment]
I think it's fixed in #1268 and #1247
